templates/etc/icinga2/conf.d/downtimes.conf.j2

```diff
@@ -4,19 +4,39 @@
  *
  *  Icinga2 service downtimes
  */
-
 {% for downtime in icinga2_master_downtimes %}
-apply ScheduledDowntime "downtime-{{ downtime.name }}" to {{ 'Service' if downtime.services else 'Host' }} {
-  comment = "{{ downtime.comment }}"
 
+
+/* {{ downtime.comment | default(downtime.name) }} */
+
+template ScheduledDowntime "template-downtime-{{ downtime.name }}" {
   ranges = {
-{% for range in downtime.ranges %}    "{{ range.day }}" = "{{ range.time }}"
-{% endfor %}  }
+{% for range in downtime.ranges %}
+    "{{ range.day }}" = "{{ range.time }}"
+{% endfor %}
+  }
+}
 
+{% if not (downtime.services | default([])) %}
+apply ScheduledDowntime "host-downtime-{{ downtime.name }}" to Host {
+  import "template-downtime-{{ downtime.name }}"
+  author = "{{ icinga2_master_downtimes_author }}"
```

Comment:
Last little nitpick: Maybe it would be nice if the author could be set per downtime. Default if not set could still be `icinga2_master_downtimes_author`.

Reply:
Implemented, thanks for the suggestion!